docs/configuration/rename_function.md

```diff
@@ -0,0 +1,86 @@
+# How to update name element in architecture object
+
+An architecture object loaded from a set of CTF-based record data or an architecture file has the elements callback, node, path, executor and topic.
+If you want to rename these elements, CARET serves the functions to update the name of the target element.
+The name of the target element in the architecture object is updated via the Python API.
+
+## Python API
+
+CARET serves Python-based APIs to update the name of the target element in an architecture object.
+
+All of the following code snippets can be executed after load environment variables with `source /path/to/ros2_caret_ws/install/setup.bash`.
+
+### Update callback name
+
+You can update callback name in architecture object from `src` to `dst` with `rename_callback` function in `Architecture` class.
+
+```python
+# arch is caret_analyze.architecture.architecture.Architecture-based object
+
+
+arch.rename_callback('src', 'dst')
+```
+
+All `callback_name` in architecture object where it is `src` are updated to `dst`.
+
+This update is also reflected in the architecture file output by [`arch.export`](./load_and_save.md#save).
+
+### Update node name
+
+You can update node name in architecture object from `src` to `dst` with `rename_node` function in `Architecture` class.
+
+```python
+# arch is caret_analyze.architecture.architecture.Architecture-based object
+
+
+arch.rename_node('src', 'dst')
+```
+
+All `node_name` in architecture object where it is `src` are updated to `dst`.
+
+This update is also reflected in the architecture file output by [`arch.export`](./load_and_save.md#save).
```

Comment:
What is 'architecture file output' ?
```suggestion
The architecture file with renamed objects is generated with an [`arch.export`](./load_and_save.md#save) command.
```

Reply:
Fix it in [42767a9](https://github.com/tier4/CARET_doc/pull/159/commits/42767a9fd0176d663e6af9f2587b5e2374840f6e).

This represent to architecture file **outputted** by arch.export().

Reply:
Two my opinion.
1. I think you should use only `by` instead of `outputted`.
2. You use `update` as noun, but in other sentence, you use it as verb. It may makes confusion.
Therefore I suggest following sentence.

```
The architecture file with renamed objects is generated by the [`arch.export`](./load_and_save.md#save) command.

Reply:
Fixed it in [3aa320b](https://github.com/tier4/CARET_doc/pull/159/commits/3aa320b36cd905375915e4a6cff8ef2ece6ae0ea).